src/components/radio-button-group/cards.stories.jsx

```diff
@@ -0,0 +1,756 @@
+				d="M2.16406 3.32812L7.88845 6.65763L2.16406 9.98711V3.32812Z"
+				fill="#3058E5"
+			/>
+			<path
+				d="M13.615 9.92969V16.5887L7.89062 13.2592L13.615 9.92969Z"
+				fill="#624AF6"
+			/>
+			<path
+				d="M13.6094 9.93105L19.3338 6.60156V13.2606L13.6094 9.93105Z"
+				fill="#3A30E5"
+			/>
+			<path
+				d="M13.6094 3.32812L19.3338 6.65763L13.6094 9.98711V3.32812Z"
+				fill="#3058E5"
+			/>
+			<path
+				d="M13.615 3.32812V9.9287L7.89062 6.59922L13.615 3.32812Z"
+				fill="#4067EE"
+			/>
+			<path
+				d="M13.6697 3.32948L7.94531 6.65899V0L13.6697 3.32948Z"
+				fill="#476EF9"
+			/>
+			<path
+				d="M7.88845 6.66016V13.3191L2.16406 9.98964L7.88845 6.66016Z"
+				fill="#624AF6"
+			/>
+			<path
+				d="M7.88845 0V6.65899L2.16406 3.32948L7.88845 0Z"
+				fill="#4067EE"
+			/>
+			<path
+				d="M7.88845 13.2578V19.9168L2.16406 16.5873L7.88845 13.2578Z"
+				fill="#7C3AED"
+			/>
+			<path
+				d="M13.6094 9.92969L19.3338 13.2592L13.6094 16.5887V9.92969Z"
+				fill="#4F46E5"
+			/>
+			<path
+				d="M2.16406 9.92969L7.88845 13.2592L2.16406 16.5887V9.92969Z"
+				fill="#814AF7"
+			/>
+			<path
+				d="M2.16406 3.32812L7.88845 6.65763L2.16406 9.98711V3.32812Z"
+				fill="#3058E5"
+			/>
+			<path
+				d="M13.615 9.92969V16.5887L7.89062 13.2592L13.615 9.92969Z"
+				fill="#624AF6"
+			/>
+		</svg>
+	);
+};
+
+const astraRadioButtonGroupData = [
+	{
+		id: '1',
+		badgeText: 'Free',
+		buttonText: 'Install',
+		svg: <AstraThemeSvg />,
+		title: 'Astra Theme',
+		description: 'Free WordPress Page Builder Plugin.',
+	},
+	{
+		id: '2',
+		badgeText: 'Free',
+		buttonText: 'Activate',
+		svg: <StartersTemplatesSvg />,
+		title: 'Starters Templates',
+		description: 'Build your dream website in minutes with AI.',
+	},
+	{
+		id: '3',
+		badgeText: 'Free',
+		buttonText: 'Install',
+		svg: <SureCartSvg />,
+		title: 'SureCart',
+		description: 'The new way to sell on WordPress.',
+	},
+	{
+		id: '4',
+		badgeText: 'Free',
+		buttonText: 'Install',
+		svg: <PrestoPlayerSvg />,
+		title: 'Presto Player',
+		description: 'Automate your WordPress setup.',
+	},
+];
+
+// Template for Type 2
+const Template2 = () => {
+	return (
+		<Container
+			containerType="flex"
+			gap="xs"
+			direction="column"
+			className="w-120 border border-solid rounded-md border-border-subtle p-4"
+		>
+			<Container.Item className="md:w-full lg:w-full">
+				<Container className="p-1" justify="between" gap="xs">
+					<Container.Item>
+						<Label className="font-semibold">
+							Extend Your Website
+						</Label>
+					</Container.Item>
+					<Container.Item
+						containerType="flex"
+						direction="row"
+						className="items-center"
+						gap="xs"
+					>
+						<Button className="p-0" variant="ghost">
+							<ArrowUpRight />
+						</Button>
+					</Container.Item>
+				</Container>
+			</Container.Item>
+			<Container.Item className="md:w-full lg:w-full bg-field-primary-background rounded-lg">
+				<Container className="grid grid-cols-2  p-1 gap-1">
```

Comment:
`className="grid grid-cols-2 p-1 gap-1"`
Instead of setting the grid directly in the className, you could use containerType="grid" along with cols={2}.
Please review other available Container props to ensure you're leveraging the full functionality.

Reply:
Please check! RadioButton background issue is also solved in this.